.github/workflows/codeql.yml

```diff
@@ -30,12 +30,25 @@ jobs:
     steps:
     - name: Checkout 🛎️
       uses: actions/checkout@v3
-
+    
+    - name: Setup Python 3.8 🐍
+      uses: actions/setup-python@v4
+      with:
+          python-version: 3.8
+          cache: 'pip'
+          
+    - name: Install dependencies 📥
+      run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          echo "CODEQL_PYTHON=$(which python)" >> $GITHUB_ENV
```

Comment:
I've disabled CodeQL from setting up the environment. I'm instead setting it up manually so that it points to Python 3.8 itself. Hence I also need to install the dependencies manually and make CodeQL use that environment instead.